TypeCobol.Analysis/Report/ZCallPgmReport.cs

```diff
@@ -0,0 +1,402 @@
+                                                    Graph.BasicBlock<Node, DfaBasicBlockInfo<Symbol>> upBlock = dfaBuilder.Cfg.AllBlocks[def.BlockIndex];
+                                                    DfaBasicBlockInfo<Symbol> upBlockData = upBlock.Data;
+                                                    for (int i = upBlockData.UseListFirstIndex; i < upBlockData.UseListFirstIndex + upBlockData.UseCount; i++)
+                                                    {
+                                                        if (dfaBuilder.UseList[i].Instruction == def.Instruction && dfaBuilder.UseList[i].Variable == sendingVariable)
+                                                        {   //(2.1.1.1.2.3) The UsePoint of the sending identifier is found then recurse compute its definition paths
+                                                            System.Text.StringBuilder newSB = new StringBuilder();
+                                                            List<Tuple<string, string>> newPtahs = ComputeUsePointDefPaths(dfaBuilder, program, dfaBuilder.UseList[i]);
+                                                            foreach (var item in newPtahs)
+                                                            {
+                                                                paths.Add(new Tuple<string, string>(item.Item1, string.Format(@"{0}<-{1}", up.Variable.FullDotName, item.Item2)));
+                                                            }
+                                                            break;
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                        }
+                                        break;
+                                    case StatementType.MoveCorrespondingStatement:
+                                        {
+                                        }
+                                        break;
+                                }
+                            }
+                            break;
+                        case CodeElementType.SetStatement:
+                            {//(2.1.1.2) a Set Statement
+                                SetStatement set = (SetStatement)def.Instruction.CodeElement;
+                                switch (set.StatementType)
+                                {
+                                    case StatementType.SetStatementForConditions://This a level 88 variable set.
+                                        {   //(2.1.1.2.1) The Set instruction is Condition variable set
+                                            //Use reflection to get sending value, because the class SetStatementForConditions is Internal to TypeCobol
+                                            System.Reflection.PropertyInfo info = set.GetType().GetProperty("SendingValue");
```

Comment:
The property is public but the class isn't 😞.
Among the derived types of `SetStatement` CodeElement, some are internal while others are public. I think all of them should be public and this would solve the case here.
Note that the reflexive call is not needed for pure Cobol (the value is in fact not currently used) but will be required for TypeCobol.

Reply:
Ok: 
https://github.com/TypeCobolTeam/TypeCobol/issues/1766

Reply:
https://github.com/TypeCobolTeam/TypeCobol/pull/1751/commits/3458bbe2f2eab46969b63733aa77b83d0b556316